scripts/sync_branding.sh

```diff
@@ -21,7 +21,7 @@ echo $mobile_root
 cd "$mobile_root"
 
 # Please update the sha when valora branding updates are needed
-valora_branding_sha=19bc30b61b48a92c2a836ce1dafa3de25775e1b3
+valora_branding_sha=9724cbeb4cb1ddf883151afa588d09a74d400ddc
```

Comment:
TODO(satish): update once https://github.com/valora-inc/valora-app-branding/pull/27 is merged